commons/src/main/java/com/ge/predix/acs/commons/web/RestErrorHandler.java

```diff
@@ -86,6 +87,9 @@ public ModelAndView createApiErrorResponse(final Exception e, final HttpServletR
         } else if (HttpMessageNotReadableException.class.isAssignableFrom(e.getClass())) {
             response.setStatus(HttpStatus.BAD_REQUEST.value());
             restApiErrorResponse.setErrorMessage("Malformed JSON syntax. " + e.getLocalizedMessage());
+        } else if (HttpMediaTypeNotSupportedException.class.isAssignableFrom(e.getClass())) {
```

Comment:
@VoollaSandeepKumar, seems like this if-else block is growing and is expected to grow more as more.  Please investigate if adding ControllerAdvice is a better approach.  For example refer to com.ge.predix.acs.ZoneDoesNotExistControllerAdvice

Reply:
As agreed, remove this file.

Reply:
Fixed